investors/models.py

```diff
@@ -120,3 +130,25 @@ class Meta:
             models.Index(fields=['-saved_at'], name='saved_saved_at_desc_idx'),
         ]
 
+class ViewedStartup(models.Model):
+    investor = models.ForeignKey(
+        'investors.Investor',
+        on_delete=models.CASCADE,
+        related_name='viewed_startups_links',
+        help_text="Investor who viewed the startup"
+    )
+    startup = models.ForeignKey(
+        "startups.Startup",
+        on_delete=models.CASCADE,
+        related_name="views"
+    )
+    viewed_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        verbose_name = "Startup view history"
+        verbose_name_plural = "Startup view histories"
+        unique_together = ("investor", "startup")
```

Comment:
unique_together constraint prevents multiple view records for the same investor/startup, which blocks history tracking. Remove unique_together or redesign if you want full view history.